### PR TITLE
Updating docs to teach how to configure MinIO

### PR DESCRIPTION
**Pull request by GustavoMaciel**
Separating the documentation of how to set up S3 in two different files, one for AWS S3 and one for MinIO.

### CODE REVIEW

File: docs/minio_s3_configuration.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+```
+The command will start the MinIO API and Console servers in ports 9000 and 9001, respectively. 
+
+Once you have successfully started the container, you can access the Console URL (usually http://127.0.0.1:9001) using the default user (username: minioadmin, password: minioadmin) to create a new user (setting up its Access Key and Secret Access Key) or create new buckets.
+
+After you finish creating a new user, remember that you'll need the proper local URL for the API to be used in the --endpoint-url parameter of the storage creation command, it'll usually be `http://127.0.0.1:9000`.
+
+`Note:` In case you decide to work with a deployed MinIO server instead, just remember to use the proper URL when creating new storage and to have your MinIO user's Access Key and Secret Access Key in hand for the following setup.
+
+# S3 bucket configuration #
```
Comment (lucasncabral) [inline docs/minio_s3_configuration.md:19]:
```suggestion
# Credentials configuration #
```

File: docs/minio_s3_configuration.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+This section explains how to configure the settings that the ML-Git uses to interact with your bucket. This requires that you have the following data:
+
+1. Profile Name
+2. Access Key ID
+3. Secret Access Key
+4. Region Name
+5. Output Format
```
Comment (lucasncabral) [inline docs/minio_s3_configuration.md:27]: To configure MinIO access credentials is it necessary to have the region and output format? I believe this is information for S3 only.

File: docs/minio_s3_configuration.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+The _Access Key ID_ and _Secret Access Key_ are the credentials for your MinIO user. The _Region Name_ identifies the Region whose servers you want to send your requests. The _Output Format_ specifies how the results are formatted.
+
+## AWS ##
```
Comment (lucasncabral) [inline docs/minio_s3_configuration.md:31]: It seems weird to me to have this `AWS` section in the MinIO steps. As well as in a few points, we have `"You can configure AWS in three ways"`. 

I understand that MiniO is an Amazon S3 compatible cloud storage server, so the way to configure the credentials is the same, but maybe adapting the text to make this configuration step clearer for MinIO will make it easier for the user understand.

File: docs/minio_s3_configuration.md (hunk 1 above)
Comment (GustavoMaciel) [reply docs/minio_s3_configuration.md]: Changed in d1aeb2a

File: docs/minio_s3_configuration.md (hunk 2 above)
Comment (GustavoMaciel) [reply docs/minio_s3_configuration.md]: Removed the Region Name in d1aeb2a, we're not sure if the MinIO actually uses or not the Output Format, so we're leaving it in for now

File: docs/minio_s3_configuration.md (hunk 3 above)
Comment (GustavoMaciel) [reply docs/minio_s3_configuration.md]: Removed the section AWS and rephrased the sentence in d1aeb2a

**Review by gilMars — APPROVED** [review]

**Review by lucasncabral — APPROVED** [review]

File: docs/minio_s3_configuration.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
+## MinIO ##
+
+[MinIO](https://min.io/) is a cloud storage server compatible with Amazon S3. The following sections will explain how to properly set up so ML-Git can work by using the Access Key and Secret Access Key of your MinIO user.
+
+### Running MinIO locally ###
+
+In case you want to run MinIO locally for testing purposes, it's possible to run a docker container using the following command:
+```
+$ docker run --name=minio --network=host minio/minio server --console-address ":9001" /data
```
Comment (andersonpablosilva) [inline docs/minio_s3_configuration.md:9]: I believe we should encourage the use of a persistent volume in the command statement (`-v /your/path:/data`) or we could mention it in the text below.

**Review by andersonpablosilva — CHANGES_REQUESTED** [review]

File: docs/minio_s3_configuration.md (hunk 4 above)
Comment (GustavoMaciel) [reply docs/minio_s3_configuration.md]: Added to command in 93c00ce

**Review by andersonpablosilva — APPROVED** [review]

File: docs/minio_s3_configuration.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+`Note:` In case you decide to work with a deployed MinIO server instead, just remember to use the proper URL when creating new storage and to have your MinIO user's Access Key and Secret Access Key in hand for the following setup.
+
+# Credentials configuration #
+
+This section explains how to configure the settings that the ML-Git uses to interact with your bucket. This requires that you have the following data:
+
+1. Profile Name
+2. Access Key ID
+3. Secret Access Key
+5. Output Format
```
Comment (gpretto) [inline docs/minio_s3_configuration.md:26]:
```suggestion
4. Output Format
```

File: docs/storage_configurations.md [hunk 6]
```diff
@@ -5,8 +5,8 @@ Currently, ML-Git supports five types of storage (S3, MinIO, Azure, GoogleDrive
 
 
 
-1. [S3](s3_configurations.md)
-2. [MinIO](s3_configurations.md)
+1. [S3](aws_s3_configuration.md)
+2. [MinIO](minio_s3_configuration.md)
```
Comment (gpretto) [inline docs/storage_configurations.md:9]:
```suggestion
1. [MinIO](minio_s3_configuration.md)
2. [S3](aws_s3_configuration.md)
```

As we want to provide the MinIO as first options for storage... I will suggest to put as first of this list.

**Review by gpretto — CHANGES_REQUESTED** [review]
Few changes and ideas.

File: docs/minio_s3_configuration.md (hunk 5 above)
Comment (GustavoMaciel) [reply docs/minio_s3_configuration.md]: Fixed in 175921d

File: docs/storage_configurations.md (hunk 6 above)
Comment (GustavoMaciel) [reply docs/storage_configurations.md]: Changed in 175921d

**Pull request merged**